pkg/inspector/image-inspector.go

```diff
@@ -359,7 +372,7 @@ func (i *defaultImageInspector) pullImage(client *docker.Client) error {
 // option's destination path.  If the destination path is empty it will write to a temp directory
 // and update the option's destination path with a /var/tmp directory.  /var/tmp is used to
 // try and ensure it is a non-in-memory tmpfs.
-func (i *defaultImageInspector) createAndExtractImage(client *docker.Client, containerName string) (*docker.Image, error) {
+func (i *defaultImageInspector) createAndExtractImage(client *docker.Client, containerName string) (*docker.Image, *docker.Container, error) {
```

Comment:
the only place where `createAndExtractImage` is being called is https://github.com/openshift/image-inspector/pull/66/files#diff-cb14c05f76ccda787aced81d2a624e69R153 where we throw away the `*docker.Container` value. Why return it at all?